{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":6804,"level":50,"msg":"[GET-ALL-CHAT-ERROR] DatabaseError [SequelizeDatabaseError]: Unknown column 'date' in 'field list'\n    at Query.formatError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:311:20)\n    at addChunk (_stream_readable.js:294:12)\n    at readableAddChunk (_stream_readable.js:275:11)\n    at Socket.Readable.push (_stream_readable.js:209:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Unknown column 'date' in 'field list'\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'date' in 'field list'\",\n    sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt` FROM `chats` AS `chats`;',\n    parameters: undefined\n  },\n  original: Error: Unknown column 'date' in 'field list'\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'date' in 'field list'\",\n    sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt` FROM `chats` AS `chats`;',\n    parameters: undefined\n  },\n  sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt` FROM `chats` AS `chats`;',\n  parameters: undefined\n}","time":"2020-03-19T02:00:29.145Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":7660,"level":50,"msg":"[GET-ALL-CHAT-ERROR] DatabaseError [SequelizeDatabaseError]: Unknown column 'createdAt' in 'field list'\n    at Query.formatError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:311:20)\n    at addChunk (_stream_readable.js:294:12)\n    at readableAddChunk (_stream_readable.js:275:11)\n    at Socket.Readable.push (_stream_readable.js:209:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Unknown column 'createdAt' in 'field list'\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'createdAt' in 'field list'\",\n    sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt` FROM `chats` AS `chats`;',\n    parameters: undefined\n  },\n  original: Error: Unknown column 'createdAt' in 'field list'\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'createdAt' in 'field list'\",\n    sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt` FROM `chats` AS `chats`;',\n    parameters: undefined\n  },\n  sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt` FROM `chats` AS `chats`;',\n  parameters: undefined\n}","time":"2020-03-19T02:05:43.223Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":6480,"level":50,"msg":"[GET-ALL-CHAT-ERROR] DatabaseError [SequelizeDatabaseError]: Unknown column 'createdAt' in 'field list'\n    at Query.formatError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:311:20)\n    at addChunk (_stream_readable.js:294:12)\n    at readableAddChunk (_stream_readable.js:275:11)\n    at Socket.Readable.push (_stream_readable.js:209:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Unknown column 'createdAt' in 'field list'\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'createdAt' in 'field list'\",\n    sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt` FROM `chats` AS `chats`;',\n    parameters: undefined\n  },\n  original: Error: Unknown column 'createdAt' in 'field list'\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'createdAt' in 'field list'\",\n    sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt` FROM `chats` AS `chats`;',\n    parameters: undefined\n  },\n  sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt` FROM `chats` AS `chats`;',\n  parameters: undefined\n}","time":"2020-03-19T02:07:34.807Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":13032,"level":50,"msg":"[GET-ALL-CHAT-ERROR] DatabaseError [SequelizeDatabaseError]: Unknown column 'createdAt' in 'field list'\n    at Query.formatError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:311:20)\n    at addChunk (_stream_readable.js:294:12)\n    at readableAddChunk (_stream_readable.js:275:11)\n    at Socket.Readable.push (_stream_readable.js:209:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Unknown column 'createdAt' in 'field list'\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'createdAt' in 'field list'\",\n    sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt` FROM `chats` AS `chats`;',\n    parameters: undefined\n  },\n  original: Error: Unknown column 'createdAt' in 'field list'\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'createdAt' in 'field list'\",\n    sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt` FROM `chats` AS `chats`;',\n    parameters: undefined\n  },\n  sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt` FROM `chats` AS `chats`;',\n  parameters: undefined\n}","time":"2020-03-19T02:08:56.803Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":4216,"level":50,"msg":"[GET-ALL-CHAT-ERROR] DatabaseError [SequelizeDatabaseError]: Unknown column 'createdAt' in 'field list'\n    at Query.formatError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:311:20)\n    at addChunk (_stream_readable.js:294:12)\n    at readableAddChunk (_stream_readable.js:275:11)\n    at Socket.Readable.push (_stream_readable.js:209:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Unknown column 'createdAt' in 'field list'\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'createdAt' in 'field list'\",\n    sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt`, `deletedAt` FROM `chats` AS `chats` WHERE (`chats`.`deletedAt` IS NULL);',\n    parameters: undefined\n  },\n  original: Error: Unknown column 'createdAt' in 'field list'\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'createdAt' in 'field list'\",\n    sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt`, `deletedAt` FROM `chats` AS `chats` WHERE (`chats`.`deletedAt` IS NULL);',\n    parameters: undefined\n  },\n  sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt`, `deletedAt` FROM `chats` AS `chats` WHERE (`chats`.`deletedAt` IS NULL);',\n  parameters: undefined\n}","time":"2020-03-19T02:13:57.570Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":13228,"level":50,"msg":"[GET-ALL-CHAT-ERROR] DatabaseError [SequelizeDatabaseError]: Unknown column 'createdAt' in 'field list'\n    at Query.formatError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Query.handler [as onResult] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:311:20)\n    at addChunk (_stream_readable.js:294:12)\n    at readableAddChunk (_stream_readable.js:275:11)\n    at Socket.Readable.push (_stream_readable.js:209:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Unknown column 'createdAt' in 'field list'\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'createdAt' in 'field list'\",\n    sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt`, `deletedAt` FROM `chats` AS `chats`;',\n    parameters: undefined\n  },\n  original: Error: Unknown column 'createdAt' in 'field list'\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Query.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'createdAt' in 'field list'\",\n    sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt`, `deletedAt` FROM `chats` AS `chats`;',\n    parameters: undefined\n  },\n  sql: 'SELECT `id`, `sender`, `message`, `date`, `createdAt`, `updatedAt`, `deletedAt` FROM `chats` AS `chats`;',\n  parameters: undefined\n}","time":"2020-03-19T02:14:29.634Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":7280,"level":50,"msg":"[CREATE-CHAT-ERROR] TypeError: Chat.createChat is not a function\n    at ChatService.createChat (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\src\\services\\chat.service.js:22:33)\n    at createChat (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\src\\controller\\chat.controller.js:13:37)\n    at C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\src\\routes\\chat.route.js:10:31\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Route.dispatch (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:281:22\n    at Function.process_params (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:335:12)\n    at next (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:275:10)\n    at Function.handle (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:174:3)\n    at router (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:47:12)\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:317:13)\n    at C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:284:7\n    at Function.process_params (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:335:12)\n    at next (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:275:10)\n    at Function.handle (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:174:3)\n    at router (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:47:12)\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:317:13)\n    at C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:284:7","time":"2020-03-19T02:19:07.876Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":11348,"level":50,"msg":"[CREATE-CHAT-ERROR] TypeError: Chat.createChat is not a function\n    at ChatService.createChat (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\src\\services\\chat.service.js:23:33)\n    at createChat (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\src\\controller\\chat.controller.js:13:37)\n    at C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\src\\routes\\chat.route.js:10:31\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Route.dispatch (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:281:22\n    at Function.process_params (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:335:12)\n    at next (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:275:10)\n    at Function.handle (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:174:3)\n    at router (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:47:12)\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:317:13)\n    at C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:284:7\n    at Function.process_params (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:335:12)\n    at next (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:275:10)\n    at Function.handle (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:174:3)\n    at router (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:47:12)\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:317:13)\n    at C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:284:7","time":"2020-03-19T02:21:19.288Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":10364,"level":50,"msg":"[CREATE-CHAT-ERROR] TypeError: Chat.createChat is not a function\n    at ChatService.createChat (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\src\\services\\chat.service.js:24:33)\n    at createChat (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\src\\controller\\chat.controller.js:13:37)\n    at C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\src\\routes\\chat.route.js:10:31\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Route.dispatch (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:281:22\n    at Function.process_params (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:335:12)\n    at next (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:275:10)\n    at Function.handle (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:174:3)\n    at router (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:47:12)\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:317:13)\n    at C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:284:7\n    at Function.process_params (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:335:12)\n    at next (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:275:10)\n    at Function.handle (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:174:3)\n    at router (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:47:12)\n    at Layer.handle [as handle_request] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:317:13)\n    at C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\express\\lib\\router\\index.js:284:7","time":"2020-03-19T02:27:27.923Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":6792,"level":50,"msg":"[CREATE-VALIDATION-ERROR] DatabaseError [SequelizeDatabaseError]: Unknown column 'createdAt' in 'field list'\n    at Query.formatError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at handler (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Prepare.onResult (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:604:11)\n    at Prepare.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:311:20)\n    at addChunk (_stream_readable.js:294:12)\n    at readableAddChunk (_stream_readable.js:275:11)\n    at Socket.Readable.push (_stream_readable.js:209:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Unknown column 'createdAt' in 'field list'\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Prepare.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'createdAt' in 'field list'\",\n    sql: 'INSERT INTO `users` (`id`,`createdAt`,`updatedAt`) VALUES (?,?,?);',\n    parameters: [\n      '4d7ae8e0-698c-11ea-ac0e-5582bf641e49',\n      '2020-03-19 02:49:50',\n      '2020-03-19 02:49:50'\n    ]\n  },\n  original: Error: Unknown column 'createdAt' in 'field list'\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Prepare.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_BAD_FIELD_ERROR',\n    errno: 1054,\n    sqlState: '42S22',\n    sqlMessage: \"Unknown column 'createdAt' in 'field list'\",\n    sql: 'INSERT INTO `users` (`id`,`createdAt`,`updatedAt`) VALUES (?,?,?);',\n    parameters: [\n      '4d7ae8e0-698c-11ea-ac0e-5582bf641e49',\n      '2020-03-19 02:49:50',\n      '2020-03-19 02:49:50'\n    ]\n  },\n  sql: 'INSERT INTO `users` (`id`,`createdAt`,`updatedAt`) VALUES (?,?,?);',\n  parameters: [\n    '4d7ae8e0-698c-11ea-ac0e-5582bf641e49',\n    '2020-03-19 02:49:50',\n    '2020-03-19 02:49:50'\n  ]\n}","time":"2020-03-19T02:49:50.467Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":6652,"level":50,"msg":"[CREATE-CHAT-ERROR] DatabaseError [SequelizeDatabaseError]: Incorrect datetime value: 'Invalid date' for column 'date' at row 1\n    at Query.formatError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Execute.handler [as onResult] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Execute.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:311:20)\n    at addChunk (_stream_readable.js:294:12)\n    at readableAddChunk (_stream_readable.js:275:11)\n    at Socket.Readable.push (_stream_readable.js:209:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Incorrect datetime value: 'Invalid date' for column 'date' at row 1\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'date' at row 1\",\n    sql: 'INSERT INTO `chats` (`id`,`sender`,`message`,`date`) VALUES (?,?,?,?);',\n    parameters: [\n      '545c6cf0-698d-11ea-b1d3-7d3b50ce78f2',\n      'ronny',\n      'hallo',\n      'Invalid date'\n    ]\n  },\n  original: Error: Incorrect datetime value: 'Invalid date' for column 'date' at row 1\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'date' at row 1\",\n    sql: 'INSERT INTO `chats` (`id`,`sender`,`message`,`date`) VALUES (?,?,?,?);',\n    parameters: [\n      '545c6cf0-698d-11ea-b1d3-7d3b50ce78f2',\n      'ronny',\n      'hallo',\n      'Invalid date'\n    ]\n  },\n  sql: 'INSERT INTO `chats` (`id`,`sender`,`message`,`date`) VALUES (?,?,?,?);',\n  parameters: [\n    '545c6cf0-698d-11ea-b1d3-7d3b50ce78f2',\n    'ronny',\n    'hallo',\n    'Invalid date'\n  ]\n}","time":"2020-03-19T02:57:11.544Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":10776,"level":50,"msg":"[CREATE-CHAT-ERROR] DatabaseError [SequelizeDatabaseError]: Incorrect datetime value: 'Invalid date' for column 'date' at row 1\n    at Query.formatError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Execute.handler [as onResult] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Execute.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:311:20)\n    at addChunk (_stream_readable.js:294:12)\n    at readableAddChunk (_stream_readable.js:275:11)\n    at Socket.Readable.push (_stream_readable.js:209:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Incorrect datetime value: 'Invalid date' for column 'date' at row 1\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'date' at row 1\",\n    sql: 'INSERT INTO `chats` (`id`,`sender`,`message`,`date`) VALUES (?,?,?,?);',\n    parameters: [\n      '2e637050-698f-11ea-9b4f-11a6776f5e4d',\n      'ronny',\n      'hallo',\n      'Invalid date'\n    ]\n  },\n  original: Error: Incorrect datetime value: 'Invalid date' for column 'date' at row 1\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'date' at row 1\",\n    sql: 'INSERT INTO `chats` (`id`,`sender`,`message`,`date`) VALUES (?,?,?,?);',\n    parameters: [\n      '2e637050-698f-11ea-9b4f-11a6776f5e4d',\n      'ronny',\n      'hallo',\n      'Invalid date'\n    ]\n  },\n  sql: 'INSERT INTO `chats` (`id`,`sender`,`message`,`date`) VALUES (?,?,?,?);',\n  parameters: [\n    '2e637050-698f-11ea-9b4f-11a6776f5e4d',\n    'ronny',\n    'hallo',\n    'Invalid date'\n  ]\n}","time":"2020-03-19T03:10:26.828Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":9000,"level":50,"msg":"[CREATE-CHAT-ERROR] DatabaseError [SequelizeDatabaseError]: Incorrect datetime value: 'Invalid date' for column 'date' at row 1\n    at Query.formatError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Execute.handler [as onResult] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Execute.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:311:20)\n    at addChunk (_stream_readable.js:294:12)\n    at readableAddChunk (_stream_readable.js:275:11)\n    at Socket.Readable.push (_stream_readable.js:209:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Incorrect datetime value: 'Invalid date' for column 'date' at row 1\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'date' at row 1\",\n    sql: 'INSERT INTO `chats` (`id`,`sender`,`message`,`date`) VALUES (?,?,?,?);',\n    parameters: [\n      '7e819f60-6991-11ea-b5bc-1725995a2a5b',\n      'ronny',\n      'Apa Kabar',\n      'Invalid date'\n    ]\n  },\n  original: Error: Incorrect datetime value: 'Invalid date' for column 'date' at row 1\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'date' at row 1\",\n    sql: 'INSERT INTO `chats` (`id`,`sender`,`message`,`date`) VALUES (?,?,?,?);',\n    parameters: [\n      '7e819f60-6991-11ea-b5bc-1725995a2a5b',\n      'ronny',\n      'Apa Kabar',\n      'Invalid date'\n    ]\n  },\n  sql: 'INSERT INTO `chats` (`id`,`sender`,`message`,`date`) VALUES (?,?,?,?);',\n  parameters: [\n    '7e819f60-6991-11ea-b5bc-1725995a2a5b',\n    'ronny',\n    'Apa Kabar',\n    'Invalid date'\n  ]\n}","time":"2020-03-19T03:27:00.237Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":10272,"level":50,"msg":"[CREATE-CHAT-ERROR] DatabaseError [SequelizeDatabaseError]: Incorrect datetime value: 'Invalid date' for column 'date' at row 1\n    at Query.formatError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:244:16)\n    at Execute.handler [as onResult] (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:51:23)\n    at Execute.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:30:14)\n    at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n    at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n    at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n    at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n    at Socket.emit (events.js:311:20)\n    at addChunk (_stream_readable.js:294:12)\n    at readableAddChunk (_stream_readable.js:275:11)\n    at Socket.Readable.push (_stream_readable.js:209:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n  name: 'SequelizeDatabaseError',\n  parent: Error: Incorrect datetime value: 'Invalid date' for column 'date' at row 1\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'date' at row 1\",\n    sql: 'INSERT INTO `chats` (`id`,`sender`,`message`,`date`) VALUES (?,?,?,?);',\n    parameters: [\n      '90954ee0-6991-11ea-a0b8-3d898a5b1527',\n      'ronny',\n      'Apa Kabar',\n      'Invalid date'\n    ]\n  },\n  original: Error: Incorrect datetime value: 'Invalid date' for column 'date' at row 1\n      at Packet.asError (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n      at Execute.execute (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n      at Connection.handlePacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:417:32)\n      at PacketParser.onPacket (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:75:12)\n      at PacketParser.executeStart (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n      at Socket.<anonymous> (C:\\Users\\NB01\\Desktop\\Final Project\\CORONA\\node_modules\\mysql2\\lib\\connection.js:82:25)\n      at Socket.emit (events.js:311:20)\n      at addChunk (_stream_readable.js:294:12)\n      at readableAddChunk (_stream_readable.js:275:11)\n      at Socket.Readable.push (_stream_readable.js:209:10)\n      at TCP.onStreamRead (internal/stream_base_commons.js:186:23) {\n    code: 'ER_TRUNCATED_WRONG_VALUE',\n    errno: 1292,\n    sqlState: '22007',\n    sqlMessage: \"Incorrect datetime value: 'Invalid date' for column 'date' at row 1\",\n    sql: 'INSERT INTO `chats` (`id`,`sender`,`message`,`date`) VALUES (?,?,?,?);',\n    parameters: [\n      '90954ee0-6991-11ea-a0b8-3d898a5b1527',\n      'ronny',\n      'Apa Kabar',\n      'Invalid date'\n    ]\n  },\n  sql: 'INSERT INTO `chats` (`id`,`sender`,`message`,`date`) VALUES (?,?,?,?);',\n  parameters: [\n    '90954ee0-6991-11ea-a0b8-3d898a5b1527',\n    'ronny',\n    'Apa Kabar',\n    'Invalid date'\n  ]\n}","time":"2020-03-19T03:27:30.562Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":5224,"level":50,"msg":"[NO-ROUTE] There is no route to /validation","time":"2020-03-19T03:40:08.514Z","v":0}
{"name":"F-SMM","hostname":"DESKTOP-2138S88","pid":2724,"level":50,"msg":"[NO-ROUTE] There is no route to /validation","time":"2020-03-19T03:40:36.695Z","v":0}
